aten/src/ATen/core/dispatch/Dispatcher.cpp

```diff
@@ -114,6 +114,8 @@ OperatorHandle Dispatcher::findOrRegisterName_(const OperatorName& op_name) {
   return handle;
 }
 
+OperatorHandle::~OperatorHandle() = default;
```

Comment:
Can you add a quick comment here linking to https://github.com/pytorch/pytorch/issues/70032? Mostly so people glancing at this code know that this was put in the C++ directly for a reason.

Reply:
Thanks, updated

Reply:
Comment should be next to line 121 rather than next to line 48 (i.e. where `Dispatcher` destructor is defined)

Reply:
Thanks, fixed :)